lib/core/versions/latest/models/ProtocolParameters.ts

```diff
@@ -14,4 +14,6 @@ export default interface ProtocolParameters {
   maxOperationsPerBatch: number;
   /** Maximum size of an operation in bytes. */
   maxOperationByteSize: number;
+  /** The factor that converts the normalized fee from blockchain into a 'per operation' fee. */
+  normalizedToPerOperationFeeFactor: number;
```

Comment:
Nit: it's long but consider renaming to `normalizedFeeToPerOperationFeeMultiplier`